gen/installer/bash.py

```diff
@@ -659,26 +659,29 @@ def do_create(tag, build_name, reproducible_artifact_path, commit, variant_argum
 
     Outputs the generated dcos_generate_config.sh as it's artifacts.
     """
-
     # TODO(cmaloney): Build installers in parallel.
     # Variants are sorted for stable ordering.
-    for variant, bootstrap_info in sorted(variant_arguments.items(), key=lambda kv: pkgpanda.util.variant_str(kv[0])):
-        print("Building installer for variant:", pkgpanda.util.variant_name(variant))
-        bootstrap_installer_name = '{}installer'.format(pkgpanda.util.variant_prefix(variant))
-        bootstrap_installer_id = all_bootstraps[bootstrap_installer_name]
-
-        installer_filename = make_installer_docker(variant, bootstrap_info['bootstrap_id'], bootstrap_installer_id)
-
-        yield {
-            'channel_path': 'dcos_generate_config.{}sh'.format(pkgpanda.util.variant_prefix(variant)),
-            'local_path': installer_filename
-        }
+    with logger.scope("building dcos_generate_config.sh"):
```

Comment:
https://github.com/dcos/dcos/blob/master/gen/installer/bash.py#L673 prints out the actual filename which will be produced in the particular variant build.